### PR TITLE
Stripe payment webhooks

### PR DESCRIPTION
**Pull request by shalapatil**
## Summary
Changed from this PR were approved and merged here https://github.com/saeloun/miru-web/pull/680
but this was reverted because stripe settings were remaining to try out on staging. 
In thie PR, there is only one line of extra change for excluding CSRF check, there I have added comment specifically for the review.

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to
      not work as expected)
- [ ] This change requires a documentation update

### Checklist:

- [ ] I have manually tested all workflows
- [ ] I have performed a self-review of my own code
- [ ] I have added automated tests for my code

### CODE REVIEW

File: app/controllers/webhooks/stripe_controller.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+class Webhooks::StripeController < ApplicationController
+  include StripeCheckout
+
+  skip_before_action :authenticate_user!
+  skip_before_action :verify_authenticity_token
+  skip_after_action :verify_authorized
+  protect_from_forgery except: :fulfill_stripe_checkout
```
Comment (shalapatil) [inline app/controllers/webhooks/stripe_controller.rb:9]: Only this line is added extra in this PR

Comment (shalapatil) [reply app/controllers/webhooks/stripe_controller.rb:9]: `protect_from_forgery except: :fulfill_stripe_checkout`

**Review by akhilgkrishnan — APPROVED** [review]
LGTM

File: Gemfile [hunk 2]
```diff
@@ -201,4 +201,5 @@ group :test do
 
   # Strategies for cleaning databases in Ruby.
   gem "database_cleaner", "~> 2.0"
+  gem "hash_dot"
```
Comment (vipulnsward) [inline Gemfile:204]: Remember to only use in specs, no usage in code

Comment (shalapatil) [reply Gemfile:204]: Yes, its added in test section only.

**Pull request merged**